### PR TITLE
[ONNX]GetShape

### PR DESCRIPTION
**Pull request by cynthia-liu**

### CODE REVIEW

**Comment by cynthia-liu** [comment]
http://10.239.47.211:18888/job/ZOO-PR-Python-Spark-2.1-py36/1527/console
Layers in Scala but not in Python: 
InternalGetShape : ./zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/layers/ReShape.scala

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/keras/layers/GetShapeSpec.scala [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+class GetShapeSpec extends KerasBaseSpec {
+
+  "GetShape" should "be test" in {
+    val seq = Sequential[Float]()
+    val input = InputLayer[Float](inputShape = Shape(3, 4), name = "input1")
+    seq.add(input)
+    val dense = Dense[Float](2, activation = "relu")
+    seq.add(dense)
+    val ss = new GetShape[Float](inputShape = Shape(3, 2))
```
Comment (zhichao-li) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/keras/layers/GetShapeSpec.scala]: change to `new GetShape[Float]()`

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/layers/ReShape.scala [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 Analytics Zoo Authors.
```
Comment (zhichao-li) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/layers/ReShape.scala]: rename to `GetShape.scala`

**Comment by zhichao-li** [comment]
jenkins?

**Comment by cynthia-liu** [comment]
ZOO-PR-Validation - Build # 1609 - Successful:

Check console output at http://10.239.47.211:18888/job/ZOO-PR-Validation/1609/ to view the results.

The pull request: https://github.com/intel-analytics/analytics-zoo/pull/1015 Successful

**Pull request merged**